cluster/evaluator/gpu-evaluator.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: minigo-gpu-evaluator-{{bnum}}-{{wnum}}
+spec:
+  parallelism: 2
+  completions: 3
+  template:
+    metadata:
+      name: gpu-evaluator
+    spec:
+      containers:
+      - name: evaluator
+        image: gcr.io/$PROJECT/$GPU_EVALUATOR_CONTAINER:$VERSION_TAG
+        imagePullPolicy: Always
+        resources:
+          limits:
+            nvidia.com/gpu: 1
+          requests:
+            nvidia.com/gpu: 1
+        volumeMounts:
+        - name: service-credentials
```

Comment:
Seems like these should be passed in as Secrets. https://kubernetes.io/docs/concepts/configuration/secret/

Reply:
does this change all the gpu-cluster-up scripts, then?

Reply:
These are now passed in as secrets.